x-pack/legacy/plugins/reporting/server/browsers/chromium/driver_factory/args.ts

```diff
@@ -61,6 +59,7 @@ export const args = ({
     flags.push('--no-sandbox');
   }
 
+  // Dead code?
```

Comment:
Looks like a bug to wrap this in a conditional. I think we want the logging flags hardcoded to "on":

https://www.chromium.org/for-testers/enable-logging